README.md

```diff
@@ -242,11 +242,9 @@ To install from a Debian Repository:
 
      For Debian-based systems like Ubuntu, configure the Debian ROCm repository as follows:
    
-      	wget -q0 – http://repo.radeon.com/rocm/apt/debian/rocm.gpg.key | 
-
-      	sudo apt-key add -echo 'deb [arch=amd64] http://repo.radeon.com/rocm/apt/debian/ xenial main' | 
-
-      	sudo tee /etc/apt/sources.list.d/rocm.list
+      	wget -qO - http://repo.radeon.com/rocm/apt/debian/rocm.gpg.key |
+        sudo sudo apt-key add -sudo sh -c 'echo deb [arch=amd64] http://repo.radeon.com/rocm/apt/debian/ xenial main 
```

Comment:
Whilst the original code is clearly wrong, this PR is not the fix. The "sudo sudo" is a major red flag.
There should be a newline between the "-" and the "echo" in the "-echo" on line 247. So the code should read

      wget -q0 – http://repo.radeon.com/rocm/apt/debian/rocm.gpg.key | 
           sudo apt-key add -

      echo 'deb [arch=amd64] http://repo.radeon.com/rocm/apt/debian/ xenial main' |
           sudo tee /etc/apt/sources.list.d/rocm.list